database/requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+-r ./requirements.txt
+
+boto3~=1.20.20
```

Comment:
to chyba nie jest dev-owa zależność, tego potrzebujemy w teorii cały czas używać, nie tylko przy developmentcie

Reply:
Ok.